README.md

```diff
@@ -353,7 +353,7 @@ example. This
 illustrates how a cache volume gets provisioned in Kubernetes using
 PMEM-CSI driver.
 
-**NOTE**: Cache volumes are local to node not Pod. If two Pods using
+**NOTE**: Cache volumes are local to node not pod. If two pods using
```

Comment:
We use "local to node" above in slightly different context (meaning not migratable in cluster) so I propose we use here different wording, like "associated with node".
Instead of two pods, lets mention multiple as this is not specifically about two.
That sentence also uses few mixed plurals/singulars and "endup". Also, I think we should use "Operators have to consider" instead of "Applications has to consider"

So what about changing to:
Cache volumes are associated with a node, not pod. Multiple pods using the same cache volume on the same node, will not get their own instance but will end up sharing the same PMEM sharing the same PMEM
volume. Operator has to consider this and use available...

Also, the link in "use available", when clicked, will show name of yaml file as "symlink", and no contents, which is not  so user-friendly as user has to navigate to that element with other means and open it (not sure how to improve this easily)

Reply:
Is it better now? I avoided the "operator" term because it is unclear whether that is a Kubernetes operator (a tool) or an administrator (a human).

Reply:
yes, looks good now, thanks!